hydra/test_utils/launcher_common_tests.py

```diff
@@ -270,9 +275,13 @@ def sweep_2_jobs(
     with sweep:
         assert sweep.temp_dir is not None
         assert sweep.returns is not None
-        temp_dir = Path(sweep.temp_dir)
 
-        multirun_cfg_path = Path(temp_dir) / "multirun.yaml"
+        if sweep.sweep_dir:
+            config_dir = Path(os.path.join(os.getcwd(), sweep.temp_dir))
```

Comment:
Try this, Path has overloading of `/` for concatenating path objects and strings.
```suggestion
            config_dir = Path.cwd() / sweep.temp_dir
```